### ISSUE TITLE
Improve the output of default and list commands

### ISSUE THREAD

**Issue by iddev5** [marler8997/zigup#35]
I suggest that default and list command to show some more details, namely:
- default command should show the actual path of master
- list command should show which one is the master
- list command should somehow mark the default compiler too

POC commit at my fork: https://github.com/iddev5/zigup/commit/b3461229a297f9d3160560dbd87542b9eacbc481

![zigup](https://user-images.githubusercontent.com/54106488/142773305-a0478819-3ca4-4217-9091-ebe9c261dfb4.png)

**Comment by marler8997** [comment]
Yeah I think this is an improvement.  I would like the ability to list just the versions as well but that should be an option the user can opt-in to rather than the default.  By default I think the output would be meant for the human not a script so it should display any useful information about each compiler in a pretty format.

**Comment by michaelbartnett** [comment]
An indicator for what Zig versions have been marked to keep would also be great to show in the list output. I've been filing bugs for the 0.10.x/0.11.0 cycle and am rapidly accumulating versions 😅
